contrib/sacrebleu/setup.py

```diff
@@ -86,7 +86,7 @@ def get_version():
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires = ['typing'],
+    install_requires = ['typing', 'numpy'],
```

Comment:
does this add too many dependencies for this lightweight project?

Reply:
I just copied it from the old chrF implemention, but you're right that it's overkill.